libraries/chain/include/eosio/chain/transaction.hpp

```diff
@@ -215,5 +215,6 @@ FC_REFLECT( eosio::chain::transaction_header, (expiration)(ref_block_num)(ref_bl
 FC_REFLECT_DERIVED( eosio::chain::transaction, (eosio::chain::transaction_header), (context_free_actions)(actions)(transaction_extensions) )
 FC_REFLECT_DERIVED( eosio::chain::signed_transaction, (eosio::chain::transaction), (signatures)(context_free_data) )
 FC_REFLECT_ENUM( eosio::chain::packed_transaction::compression_type, (none)(zlib))
-// @ignore unpacked_trx
-FC_REFLECT( eosio::chain::packed_transaction, (signatures)(compression)(packed_context_free_data)(packed_trx) )
+
```

Comment:
Why is unpacked_trx not needed to be ignored anymore?

Reply:
I restored this ignore + one other that was required.  The reason that the lack of @ignore worked was due to an error in the regex in the validation script which caused the last FC_REFLECT macro in a file to not be checked.